packages/react-dom-bindings/src/server/ReactFizzConfigDOMLegacy.js

```diff
@@ -194,6 +193,10 @@ export function getViewTransitionFormatContext(
   return parentContext;
 }
 
+export function canHavePreamble(): boolean {
+  return false;
```

Comment:
@gnoff Top level Suspense creates a scenario where you sometimes have to emit some kind of top level HTML tag to force Suspense boundaries to flush early. This shouldn't be necessary in `renderToString` which is a legacy API.

I think I can basically disable the feature this way.